### PR TITLE
[Fix #1445][Fix #1401] Display Job Metrics per channel.

### LINKED ISSUE #1401: Add help text for what “Adjusted” metrics mean under Jobs section
Hi @glogiotatidis 

As we have more team members reviewing metrics, let's have a help text to define what our Adjusted impressions mean in comparison to regular impressions. 

Screen Shot 2020-06-23 at 12 49 55 PM

### LINKED ISSUE #1445: Pull channel metrics from telemetry for each job

### PR DESCRIPTION
**Pull request by glogiotatidis**
## Description
Extracts per channel performance numbers from `JobDailyPerformace` objects for each `Job`. Removes a few `JobDailyPerformance` convenience fields to simplify the db schema.

## Tasks
- [ ] Populate `JobDailyPerformance.adj_impression_percentrage` and `JobDailyPerformance.adj_client_percentrage` after deploying

### CODE REVIEW

**Review by pmac — APPROVED** [review]
Looks good to me. I'm taking your word for it that the cool custom query and stuff is working the way you want. I made some comments but you're free to take or leave it.

File: snippets/base/templates/admin/metrics_table.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+ #metrics-table td {
+     text-align: right;
+ }
+
+ #metrics-table th {
+     background-color: #e3e3e3;
+ }
+</style>
+<table id="metrics-table">
```
[the remark this answers is not in the document]
Comment (glogiotatidis) [reply snippets/base/templates/admin/metrics_table.html:19]: Good point, will rewrite

[the remark this answers is not in the document]
Comment (glogiotatidis) [reply snippets/base/templates/admin/metrics_table.html]: Changed to `class`

**Comment by glogiotatidis** [comment]
Ευχαριστώ!

**Pull request merged**